graphql/tests/query/patch/queries/patch.graphql

```diff
@@ -2,6 +2,7 @@
   patch(patchId: "5dd2e89cd1fe07048e43bc8f") {
     id
     author
+    generatedTaskCounts
```

Comment:
nit. since `generatedTaskCounts` is a subresolver, I think the test would fit slightly better as a new folder at `graphql/tests/patch/generatedTaskCounts`